### ISSUE TITLE
Make the title of the stop departures page into a heading

### ISSUE THREAD

**Issue by akaplo** [umts/pvta-multiplatform#325]
"Haigis Mall (74)" isn't read by Aria as a heading - it should be.

**Comment by akaplo** [comment]
This was closed in #330, #woops.
